.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -18,3 +18,11 @@ assignees: ''
 
 ## Additional context
 <!-- Add any other context or screenshots about the feature request here. -->
+
+## Don't forget to add labels!
+- component label (`neofs-adm`, `neofs-storage`, ...)
+- issue type (`enhancement`, `refactor`, ...)
+- `goodfirstissue`, `helpwanted` if needed
```

Comment:
about the triage?

Reply:
`triage` is added by default to all newly created issuses

Reply:
i meant removing it if it is already discussed. but yes, forget about the default behavior